templates/common/infra/terraform/core/database/cosmos/cosmos_variables.tf

```diff
@@ -0,0 +1,19 @@
+variable "location" {
+  description = "The supported azure location where the resource deployed"

```

Comment:
Please scrub for any lower case azure and change to Azure, thanks

Reply:
fixed